Marlin/Marlin_main.cpp

```diff
@@ -9276,6 +9286,22 @@ inline void gcode_M999() {
   }
 #endif
 
+#if ENABLED(SINGLENOZZLE_SWITCHING_EXTRUDER) 
+  inline void move_extruder_servo(uint8_t e) {
+  #if EXTRUDERS == 2
+    const int angles_e1_e2[2] = SWITCHING_EXTRUDER_SERVO_E1_E2_ANGLES;
+    MOVE_SERVO(SWITCHING_EXTRUDER_SERVO_E1_E2_NR, angles_e1_e2[e]);
+  #else
+    const int angles_e1_e2[2] = SWITCHING_EXTRUDER_SERVO_E1_E2_ANGLES;
+    const int angles_e3_e4[2] = SWITCHING_EXTRUDER_SERVO_E3_E4_ANGLES;
+    if (e < 2) 
+      MOVE_SERVO(SWITCHING_EXTRUDER_SERVO_E1_E2_NR, angles_e1_e2[e]);
+    else      
+      MOVE_SERVO(SWITCHING_EXTRUDER_SERVO_E3_E4_NR, angles_e3_e4[e - 2]);
+  #endif
```

Comment:
Indent everything here from `#if` to `#endif` by 2 spaces.
http://marlinfw.org/docs/development/coding_standards.html